README.md

```diff
@@ -85,6 +85,13 @@ object_client.metadata.dublin_core
 
 # Get the public descriptive XML representation
 object_client.metadata.descriptive
+
+# Return the Cocina metadata
+object_client.find
+
+# Query for an objects collections
+object_client.queries.collections
```

Comment:
Not a show stopper for me, but don't love `queries`. Everything is a query. Or nothing is a query.

Reply:
Except the things that are writes.  I can get behind moving find to `Queries` if you like that.

Reply:
I don't feel strongly; defer to your judgment.

Reply:
I've relented and done it the jlitt way.